Library/ViewModels/ChangeEmailViewModelTests.swift

```diff
@@ -94,21 +94,33 @@ final class ChangeEmailViewModelTests: TestCase {
     }
   }
 
-  func testOnePasswordButtonHidesIfNotAvailable() {
+  func testOnePasswordButtonHidesProperly_OnIOS11AndEarlier() {
     self.vm.inputs.viewDidLoad()
-    self.vm.inputs.onePassword(isAvailable: false)
 
-    self.onePasswordButtonIsHidden.assertValues([true])
+    let iOS12: (Double) -> Bool = { _ in false }
+    withEnvironment(isOSVersionAvailable: iOS12) {
```

Comment:
Is the one password extension dependent on the availability API? It seems that the test here is behaving independently of the OS version.

Reply:
Ah now I see, on iOS 11 and under it just returns the value of the `onePasswordIsAvailableProperty` signal.